go-controller/pkg/ovn/master.go

```diff
@@ -625,6 +625,15 @@ func (oc *Controller) ensureNodeLogicalNetwork(nodeName string, hostsubnet *net.
 		return err
 	}
 
+	// Add any service reject ACLs applicable for TCP LB
+	acls := oc.GetAllACLsForServiceLB(oc.TCPLoadBalancerUUID)
+	if len(acls) > 0 {
+		_, _, err = util.RunOVNNbctl("add", "logical_switch", nodeName, "acls", strings.Join(acls, ","))
+		if err != nil {
+			klog.Warningf("Unable to add reject ACLs: %s for switch: %s", acls, nodeName)
```

Comment:
can you please capture `err` value in the Warning message so that we will know exactly as to why things failed?

Reply:
yes